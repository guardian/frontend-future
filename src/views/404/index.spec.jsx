// @flow

import test from 'utils/test-render';
import FourOhFour from './index';

jest.mock('./style.scss', () => ({
    message: {
        color: 'hotpink',
    },
}));

describe('404', () => {
    test(<FourOhFour />);
});
