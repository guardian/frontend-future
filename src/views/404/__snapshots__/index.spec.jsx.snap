// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 generates the expected markup 1`] = `
Object {
  "css": "<style class=\\"_styletron_hydrate_\\">.l:after{content:\\" | \\"}.a{margin-top:20px}.b{border-top-width:3px}.c{border-color:#0061A6}.d{border-top-style:solid}.e{padding-top:2px}.f{font-size:11px}.g{line-height:14px}.h{margin:15px 0}.i{clear:both}.j{font-family:arial}.k{display:inline}.m{color:#005689}.n{text-decoration:none}</style>",
  "html": "<div><div><a href=\\"/\\">Home</a></div><svg><text>test svg</text></svg><h1>Sorry - we haven’t been able to serve the page you asked for.</h1><div class=\\"a\\"><div><h2>404</h2></div><p>You may have followed a broken or outdated link, or there may be an error on our site.</p><p>Please follow one of the links below to continue exploring.</p><nav><ul role=\\"navigation\\"><li><a href=\\"/\\" target=\\"_self\\">home</a></li><li><a href=\\"/uk\\" target=\\"_self\\">UK</a></li><li><a href=\\"/world\\" target=\\"_self\\">world</a></li><li><a href=\\"/sport\\" target=\\"_self\\">sport</a></li><li><a href=\\"/football\\" target=\\"_self\\">football</a></li><li><a href=\\"/commentisfree\\" target=\\"_self\\">comment</a></li><li><a href=\\"/culture\\" target=\\"_self\\">culture</a></li><li><a href=\\"/business\\" target=\\"_self\\">economy</a></li><li><a href=\\"/lifeandstyle\\" target=\\"_self\\">life</a></li><li><a href=\\"/fashion\\" target=\\"_self\\">fashion</a></li><li><a href=\\"/environment\\" target=\\"_self\\">environment</a></li><li><a href=\\"/technology\\" target=\\"_self\\">tech</a></li><li><a href=\\"/money\\" target=\\"_self\\">money</a></li><li><a href=\\"/travel\\" target=\\"_self\\">travel</a></li><li><a href=\\"https://soulmates.guardian.co.uk/\\" target=\\"_self\\">soulmates</a></li><li><a href=\\"http://m.jobs.guardian.co.uk/\\" target=\\"_blank\\">jobs</a></li><li><a href=\\"/guardian-masterclasses\\" target=\\"_blank\\">masterclasses</a></li></ul></nav></div><div class=\\"b c d e f g h i\\"><ul class=\\"j\\"><li class=\\"k l\\"><a href=\\"/help\\" class=\\"m n\\">Help</a></li><li class=\\"k l\\"><a href=\\"/help/contact-us\\" class=\\"m n\\">Contact us</a></li><li class=\\"k l\\"><a href=\\"https://www.surveymonkey.com/s/theguardian-beta-feedback\\" target=\\"_blank\\" class=\\"m n\\">Feedback</a></li><li class=\\"k l\\"><a href=\\"/help/terms-of-service\\" class=\\"m n\\">Terms &amp;amp; conditions</a></li><li class=\\"k l\\"><a href=\\"/help/privacy-policy\\" class=\\"m n\\">Privacy policy</a></li></ul><p class=\\"j\\"><small>© Guardian News and Media Limited or its affiliated companies. All rights reserved. Registered in England and Wales. No. 908396. Registered office: PO Box 68164, Kings Place, 90 York Way, London N1P 2AP</small></p></div></div>",
}
`;
